.github/workflows/zephyr_build.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+name: Build samples with Twister
```

Comment:
I would suggest putting Zephyr in this line, maybe "Build Zephyr samples with Twister". It took me a little searching to find this build, as I was expecting to see Zephyr.

Reply:
@PerMac I think that comment is reasonable.

Reply:
done